generator/internal/scaffold_generator.cc

```diff
@@ -668,15 +665,9 @@ can override the default policies.
 
 [cloud-service-docs]: https://cloud.google.com/$site_root$
 [exponential backoff]: https://en.wikipedia.org/wiki/Exponential_backoff
-[gcs-quickstart]: https://cloud.google.com/$site_root$/docs/quickstarts 'Quickstarts'
```

Comment:
less important, but these changes could be propagated too

Reply:
Done.